test/controllers/EditSetControllerSpec.scala

```diff
@@ -48,868 +52,81 @@ class EditSetControllerSpec extends BaseSpec {
+      ).thenReturn(getEditSetDisplay)
       val editSet = controller
         .view("1")
         .apply(
           CSRFTokenHelper.addCSRFToken(
-            FakeRequest(GET, "/edit-set/1")
-              .withSession(SessionKeys.token -> validSessionToken)
+            FakeRequest(GET, "/edit-set/1").withSession(SessionKeys.token -> validSessionToken)
           )
         )
 
       status(editSet) mustBe OK
       contentType(editSet) mustBe Some("text/html")
-      val document = asDocument(editSet)
-      document must haveCaption("Showing 1 - 10 of 12 records")
-      document must haveHeader("Edit set: COAL 80 Sample")
-    }
-
-    "render the edit set page from the router" in {
-      val request = FakeRequest(GET, "/edit-set/1").withSession(SessionKeys.token -> validSessionToken)
-      val editSet = route(app, request).get
-
-      status(editSet) mustBe OK
-      contentType(editSet) mustBe Some("text/html")
-      val document = asDocument(editSet)
-      document must haveCaption("Showing 1 - 10 of 12 records")
-      document must haveHeader("Edit set: COAL 80 Sample")
-    }
-
-    "all ids in the document conform to w3c reccomendations" in {
-      val request = FakeRequest(GET, "/edit-set/1").withSession(SessionKeys.token -> validSessionToken)
-      val editRecordPage = route(app, request).get
-
-      status(editRecordPage) mustBe OK
-      asDocument(editRecordPage) must haveAllLowerCaseIds
-    }
-
-    "all class names in the document conform to w3c reccomendations" in {
-      val request = FakeRequest(GET, "/edit-set/1").withSession(SessionKeys.token -> validSessionToken)
-      val editRecordPage = route(app, request).get
-
-      status(editRecordPage) mustBe OK
-      asDocument(editRecordPage) must haveAllLowerCssClassNames
-    }
-
-    "redirect to the login page from the application when requested with invalid session token" in {
-      val controller = inject[EditSetController]
-      val editSet = controller
-        .view("1")
-        .apply(
-          FakeRequest(GET, "/edit-set/1")
-            .withSession(SessionKeys.token -> invalidSessionToken)
-        )
-
-      status(editSet) mustBe SEE_OTHER
-      redirectLocation(editSet) mustBe Some("/login")
-    }
-
-    "redirect to the login page from the router when requested with invalid session token" in {
-      val request = FakeRequest(GET, "/edit-set/1").withSession(SessionKeys.token -> invalidSessionToken)
-      val editSet = route(app, request).get
-
-      status(editSet) mustBe SEE_OTHER
-      redirectLocation(editSet) mustBe Some("/login")
     }
 
     "order records" when {
 
       def orderingRequest(field: String, direction: String, offset: Int = 1) = {
         val request = FakeRequest(GET, s"/edit-set/1?field=$field&direction=$direction&offset=$offset")
           .withSession(SessionKeys.token -> validSessionToken)
-        route(app, request).get
-      }
-
-      "CCR, ascending" in {
```

Comment:
I thought that @rwalpole asked for all the scenarios to be maintained.

But many have been deleted.

Reply:
These sorting tests would still seem to have value as the sorting occurs within the controller.

Reply:
Added all the tests for sorting and the invalid session token but not w3c reccomendations due to mockeditSet

Reply:
Confirmed.